### PR TITLE
ex: adds usage examples for Hacktoberfest

### PR DESCRIPTION
**Pull request by lizgehret**
This PR is part of our 2022 Hacktoberfest team-wide sprint to add usage examples to all methods in each of our core distribution plugins. The methods examples have been added for are as follows:

```
denoise single
denoise paired
```

### CODE REVIEW

File: q2_dada2/tests/test_denoise.py [hunk 1]
```diff
@@ -24,6 +24,13 @@ def _sort_seqs(seqs):
     return sorted(list(seqs), key=lambda x: x.metadata['id'])
 
 
+class TestBase(TestPluginBase):
```
Comment (ChrisKeefe) [inline q2_dada2/tests/test_denoise.py:27]: Is TestBase the best name for this?

Comment (lizgehret) [reply q2_dada2/tests/test_denoise.py:27]: Good question @ChrisKeefe - I pulled this from the formatting of our other unit tests (I've put usage examples in there before along with the testing set up) for future usability if any factories need to be initialized there, but I'm not attached to it - I could change it to TestExamples instead. Or add the test_examples method under the TestUtils() class?

Comment (lizgehret) [reply q2_dada2/tests/test_denoise.py]: For posterity - we discussed offline and I have updated this class to TestExamples()

**Review by Oddant1 — APPROVED** [review]
Looks good to me

**Pull request merged**